shotover-proxy/src/protocols/cassandra_protocol2.rs

```diff
@@ -913,46 +935,4 @@ mod cassandra_protocol_tests {
         }];
         test_frame_codec_roundtrip(&mut codec, &bytes, messages);
     }
```

Comment:
test_process_cassandra_frame got removed. Does this get covered elsewhere?

Reply:
https://github.com/shotover/shotover-proxy/pull/356#discussion_r757148815 that was answered here, right?

Reply:
lol I forgot I had already looked at it apologies!